decentraland/unity-explorer
# feat: asset bundle fallback

## What does this PR change?


- Makes the scene definition to be fetched from the AB registry instead of the Catalyst Active Endpoints
- Behind feature flag

## How to test the changes?

1. The first test phase should be done in `zone`. 
2. Create a scene using the builder and deploy it
3. Open the client using `--dclenv zone`
4. Go to you scene, you should see it and see this message correctly
![image](https://github.com/user-attachments/assets/8e6edceb-d394-4e59-9ede-02277118ed07)

6. Using the builder, create a broken state. IE: delete all images so the ABConverter fails and add a new one. Deploy it
7. Go back to the client. You should still be able to see it and see this message correctly
![image](https://github.com/user-attachments/assets/53b6ed62-10a0-4993-afca-03b99b7d6e1d)


----------------------------------------------------------

1. The second phase is the happy path in org. Open and walk around. See that scenes, roads and LODs load correctly
2. Go to `-129,-77`. Debug message should look like this

![image](https://github.com/user-attachments/assets/8e6edceb-d394-4e59-9ede-02277118ed07)


----------------------------------------------------------

1. The third step is the feature flag. Onces finished the two tests above, let me know and I ll disable the feature flag. We then do the happy path. Everything should work as expected


## Considerations

- This change is only for scenes in Genesis City. The registry is not in use for worlds, wearables or emotes
- As of 27/01, the following list of 282 active scenes don't have at least one working ABs. This scenes/lods wont be renderer

[CatalystRegistryDifferences.txt](https://github.com/user-attachments/files/18560446/CatalystRegistryDifferences.txt)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
